### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by lresende**

### CODE REVIEW

File: kernel_gateway/_version.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 version_info = (
     2,
-    4,
+    5,
```
Comment (rolweber) [inline kernel_gateway/_version.py]: Shouldn't this version remain on 2.4.0 until the release has been tagged?

Comment (rolweber) [reply kernel_gateway/_version.py]: Never mind... the 'dev' below needs to be removed for the release build anyway, so the version number can be fixed on the same occasion.

**Comment by kevin-bates** [comment]
I suspect Luciano forgot to push the tag relative to https://github.com/jupyter/kernel_gateway/pull/324/commits/ba9296fefd80dcdcc6560bd8598f2c18cab4cc68

**Comment by rolweber** [comment]
Ah, I see the tag now: https://github.com/jupyter/kernel_gateway/releases
I probably just missed that yesterday, looking only at the PR.

**Pull request merged**